tests/dragonfly/cluster_test.py

```diff
@@ -162,12 +168,11 @@ async def test_cluster(df_local_factory):
     c_masters = [aioredis.Redis(port=master.port) for master in masters]
     c_masters_admin = [aioredis.Redis(port=master.admin_port) for master in masters]
 
-    master_ids = []
-
-    for i in range(len(masters)):
-        id = (await c_masters_admin[i].execute_command("DFLYCLUSTER MYID")).decode()
-        print(f'Master {i} id is {id}')
-        master_ids.append(id)
+    master_ids = [
+        id.decode()
+        for id in await asyncio.gather(*(c_admin.execute_command("DFLYCLUSTER MYID")
```

Comment:
you can write a small function calling execute_command("DFLYCLUSTER MYID") and returning the decoded result. 
asyncio.gather with this function

Reply:
I'm far for an expert on Python `async`, so I'm probably wrong. But I was under the impression that `gather` works on futures / async stuff, and so I'll need to pass in something of that family.
Now, `decode()` is a regular (sync) function, so how would that work?

```python
(async execute_command("DFLYCLUSTER MYID")).decode()
```
Is no longer async right?

Reply:
This does not work?
```
async def get_my_id(client):
  id = await execute_command("DFLYCLUSTER MYID")
  return id.decode()
```

Reply:
It worked! Thanks :)